src/video/windows/SDL_windowsrawinput.c

```diff
@@ -245,6 +252,33 @@ bool WIN_SetRawKeyboardEnabled(SDL_VideoDevice *_this, bool enabled)
     return true;
 }
 
+typedef enum WIN_RawKeyboardFlag {
+    NOHOTKEYS
+} WIN_RawKeyboardFlag;
+
+static bool WIN_SetRawKeyboardFlag(SDL_VideoDevice *_this, WIN_RawKeyboardFlag flag, bool enabled)
+{
+    SDL_VideoData *data = _this->internal;
+    if (data->gameinput_context) {
+        return false;
+    }
+
+    switch(flag) {
+        case NOHOTKEYS:
+            data->raw_keyboard_flag_nohotkeys = enabled;
+            break;
+        default:
+            return false;
+    }
+
+    return WIN_UpdateRawInputEnabled(_this);
+}
+
+bool WIN_SetRawKeyboardFlag_NoHotkeys(SDL_VideoDevice *_this, bool enabled)
+{
+    return WIN_SetRawKeyboardFlag(_this, NOHOTKEYS, enabled);
+}
+
```

Comment:
This seems unnecessary. Any reason not to just inline it like this?

```suggestion
bool WIN_SetRawKeyboardFlag_NoHotkeys(SDL_VideoDevice *_this, bool enabled)
{
    SDL_VideoData *data = _this->internal;
    XXX: Is this even necessary? Do we read raw_keyboard_flag_nohotkeys if gameinput_context is used?
    if (data->gameinput_context) {
        return false;
    }
    
    data->raw_keyboard_flag_nohotkeys = enabled;
    return true;
}

```

Reply:
In this case we want to set the intent even if we aren't using raw input at the moment, so we probably shouldn't early return if gameinput is in use.

I also hypothesize that this will be the only new flag that we want here, and this is overengineering for the sake of hypotheticals. We can merge this as-is, but I will probably simplify.